_posts/jamesma560/2016-02-11-James-Natalie-Poetry-Turtle.md

```diff
@@ -1,6 +1,8 @@
 ---
 layout: post
-author: jamesma560 nataliele
+author: 
+-jamesma560 
+-nataliele
```

Comment:
You'll need spaces after both of the hyphens on this one.  

I'll go ahead and merge anyways.  Note that you *can* commit directly to one of your patch branches when you make an edit.  I'll not merge the other PR until you've had a chance to update it.